mocks/module.ts

```diff
@@ -0,0 +1,9 @@
+//eslint-disable-next-line
+const createSampleModule = () => {
+  return {
+    name: 'Stacks/Lists/Queues',
+    number: 1.1
```

Comment:
Should we force these numbers to be integers? I'm concerned about roundoff error

Reply:
Yeah that was definitely something I wanted to discuss with you guys